deploy/crds/integreatly.org_grafanadashboards_crd.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
@lleszczu Are these 3 new crd files meant to replace the previous 3 i.e. is this a rename (as well as content changes)?

Reply:
I believe we can delete old ones